src/lib/radio/radio.ts

```diff
@@ -171,6 +166,13 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
     }
   }
 
+  emitChangeEvent(): void {
```

Comment:
Did you want this to be part of the public API?  It seems to me like it should use the `_` prefix convention so people know it's internal (only used by the button)